database/database.js

```diff
@@ -1,7 +1,7 @@
 const mongoose = require('mongoose');
 
 mongoose.connect(
-  'mongodb://localhost/vourcher',
+  'mongodb://172.17.0.2/vourcher',
```

Comment:
Is '172.17.0.2' the hostname that you chose, for storing your DB on Amazon cloud?

Reply:
172.17.0.2 corresponds to the IP within the EC2 instance that let's my microservice's container connect to the mongodb database that is also within that same EC2 instance.